cmd/prow-job-dispatcher/main.go

```diff
@@ -529,25 +529,59 @@ func clustersMapToSet(clusterMap ClusterMap) sets.Set[string] {
 	return clusterSet
 }
 
-func createPR(o options) {
+func gitCloneRelease() error {
+	cmd := exec.Command("git", "clone", "--depth", "1", "--single-branch", "git@github.com:openshift/release.git")
+
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to clone repository: %w, output: %s", err, string(output))
+	}
+
+	return nil
+}
+
+func cleanup(directory string) {
+	err := os.RemoveAll(directory)
+	if err != nil {
+		logrus.WithField("directory", directory).WithError(err).Error("failed to remove directory")
+	}
+	logrus.WithField("directory", directory).Info("Successfully removed directory")
+}
+
+func createPR(o options, config *dispatcher.Config) {
+	targetDirWithRelease := filepath.Join(o.targetDir, "/release")
+	cleanup(targetDirWithRelease)
```

Comment:
Did you mean to call this right away? I would have expected it only in the next line

Reply:
if garbare from other times/restarts etc